### ISSUE TITLE
Inertia tensor to quaternion bug

### ISSUE THREAD

**Issue by odydoum** [siemens/ros-sharp#259]
* [x] I am at the right place and my issue is directly related to ROS#. General technical questions I would post e.g. at [ROS Answers](https://answers.ros.org/) or [Stack Overflow](https://stackoverflow.com). For library-specific questions I would look for help in the corresponding library forums.
* [x] I have thoroughly read [the Contributing Guideline](Contributing.md) and writing this issue is the right thing to do in my case.

## I found a bug! ##
* [x] I am using the latest ROS# version available here on the master branch.
* [x] I am adding all required information, code and data files, screenshots and log files so that you can reproduce the problem.

**Here is my bug description:**
I believe there is a mistake in method
```
private static Quaternion UrdfInertial.ToQuaternion(Vector3 eigenvector0, Vector3 eigenvector1, Vector3 eigenvector2) 
{
            //From http://www.euclideanspace.com/maths/geometry/rotations/conversions/matrixToQuaternion/
```
that results in wrong inertia tensor values when importing from a urdf file. The algorithm presented in the site takes as input a rotation matrix. The eigenvectors should form this rotation matrix, where each eigenvector is a column. However, for example

```
qw = (eigenvector1[0] - eigenvector0[1]) / s;    // line 170
// should correspond to qw = (m10 - m01)/s  where m the rotation matrix, according to the website
// actually corresponds to qw = (m01 - m10)/s because eigenvectorX refers to X column and the index to the row of m
```
So basically, it is producing the quaternion of the transpose rotation from the one that is supposed to be.

Also
```
float s = Mathf.Sqrt((tr + 1.0f) * 2f);   // line 145     
```
should be
```
float s = Mathf.Sqrt(tr + 1.0f) * 2f;   // line 145                
```

**Comment by MartinBischoff** [comment]
Hi @odydoum 

thank you for digging into this problem and for reporting such a precise solution here!

Would you please review my pull request with the changes you suggested above: https://github.com/siemens/ros-sharp/pull/260 

Please feel free to directly make a pull request with your fixes next time.
